database/migrations/001_init_database.go

```diff
@@ -120,8 +120,9 @@ var Migration001InitDatabase = &migrate.Migration{
 			builder_pubkey  varchar(98) NOT NULL,
 			description    	text NOT NULL,
 
-			is_high_prio    boolean NOT NULL,
-			is_blacklisted  boolean NOT NULL,
+			status             int NOT NULL,
+			collateral_value   NUMERIC(48, 0),
+			collateral_id      bigint NOT NULL,
```

Comment:
you'd want to create a new migration file applies the changes to the database schema.

Reply:
got it. figured that might be the case. leaving this thread open as i work through that! thanks!

Reply:
alright. i see that the code is outdated, but this is not yet done in the code. All existing migrations need to be left untouched, and a single new migration file created that modifies the previous database. Take a look at this migration for reference: https://github.com/flashbots/mev-boost-relay/blob/main/database/migrations/002_bid_remove_isbest_add_receivedat.go

Reply:
thanks, Chris. i took a swing at this here: https://github.com/michaelneuder/mev-boost-relay/pull/2/commits/c4bf788bc5468d7c8c7c50dbbfc155550eac2b6f. still need to test this to make sure the DB changes make sense, but i think this overall approach of creating the new columns, updating the column values based on the `is_high_prio` and `is_blacklisted` columns, and then dropping those two should be a good approach. let me know if i am missing something here! thanks again :-)